infra/modules/providers/azure/service-plan/README.md

```diff
@@ -18,33 +18,37 @@ Please click the [link](https://www.terraform.io/docs/providers/azurerm/r/app_se
 
 ## Usage
 
+### Module Definition
+
+Service Plan Module : infra/modules/providers/azure/service-plan
+
 ```
-variable "name" {
-  default = "prod"
+variable "resource_group_name" {
+  value = "test-rg"
 }
 
-variable "location" {
-  default = "eastus"
+variable "resource_group_location" {
+  value = "eastus"
 }
 
-resource "azurerm_resource_group" "svcplan" {
-  name     = "${var.resource_group_name}"
-  location = "${var.resource_group_location}"
-  tags     = "${var.resource_tags}"
+variable "service_plan_name" {
+  value = "test-svcplan"
 }
 
-resource "azurerm_app_service_plan" "svcplan" {
+module "service_plan" {
   name                = "${var.service_plan_name}"
-  location            = "${azurerm_resource_group.svcplan.location}"
-  resource_group_name = "${azurerm_resource_group.svcplan.name}"
-  kind                = "${var.service_plan_kind}"
-  tags                = "${var.resource_tags}"
-  reserved            = "${var.service_plan_kind == "Linux" ? true : "${var.service_plan_reserved}"}"
-
-  sku {
-    tier      = "${var.service_plan_tier}"
-    size      = "${var.service_plan_size}"
-    capacity  = "${var.service_plan_capacity}"
-  }
+  location            = "${var.resource_group_location}"
+  resource_group_name = "${var.resource_group_name}"
```

Comment:
I believe you’re missing the source property

Reply:
I believe we agreed the "source property" is something that is needed in azure-simple while my documentation needs location to "module definition" as described above. 
My source link is going to be current directory. But for the user, it can also be a github link depending on how they want to use it.
That will be part of Azure-simple documentation on how the module was sourced.

Reply:
Assuming the reader knows little about terraform, you may want to consider adding `source = "github.com/Microsoft/cobalt/infra/modules/providers/azure/service-plan"` so it's more clear to the reader where this dependency is being referenced.